### PR TITLE
New feature - send custom message

### PR DESCRIPTION
**Pull request by asafov**
Use:
set('telegram_custom_text', 'Custom message test');
invoke('telegram:notify:custom');

| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | N/A

### CODE REVIEW

**Comment by asafov** [comment]
Merge please. Only new function

File: contrib/telegram.php [hunk 1]
```diff
@@ -124,3 +124,37 @@
...
+    if (!get('telegram_chat_id', false)) {
+        return;
+    }
+
+    if (get('telegram_custom_text', '') == '') {
+        return;
+    }
+
+    $telegramUrl = get('telegram_url') . '?' . http_build_query (
+            Array (
```
Comment (antonmedv) [inline contrib/telegram.php:143]: Plae use `[]` in new code.

File: contrib/telegram.php [hunk 2]
```diff
@@ -124,3 +124,37 @@
...
+
+    if (get('telegram_proxy', '') !== '') {
+        $httpie = $httpie->setopt(CURLOPT_PROXY, get('telegram_proxy'));
+    }
+
+    $httpie->send();
+})
+    ->once()
+    ->shallow()
+    ->hidden();
```
Comment (antonmedv) [inline contrib/telegram.php:160]: And a new line here. I think some lint before tests will be cool.

**Comment by antonmedv** [comment]
I'm closing this PR for now in order to housekeeping. Fill free to reopen if needed.

**Pull request closed without merging**